CK2Plus_expanded/history/provinces/585 - Gorodez.txt

```diff
@@ -0,0 +1,35 @@
+# 585 - Gorodez
+
+# County Title
+title = c_gorodez
+
+# Settlements
+max_settlements = 3
+
+b_gorodez = tribal	#1172
+#b_kitezh			#legendary city "founded" between 1188 & 1238
+
+#b_bor			#XIVth
+#b_puchishche	#1594
+#b_tarasikha
+#b_chistoye_pole
+#b_spasskoye
+
+culture = mari
+religion = finnish_pagan
+terrain = plains
+
+# History
+988.1.1 = {
+	religion = orthodox
+}
+1087.1.2 = {
+	culture = russian
+}
+1172.1.1 = {
+	b_gorodez = castle
+	b_kitezh = city

```

Comment:
If this was founded between 1188 and 1238, how come we have it being founded in 1172?

Reply:
From what I can see, it looks like the castle started as a fortress (likely a tribal village for our purposes) in 1152.

Reference: https://en.wikipedia.org/wiki/Gorodets,_Nizhny_Novgorod_Oblast

Reply:
I'm unsure why it's even included in the history files, based on what I'm reading here. @daniloy I assume that barony is vanilla?

Reference: https://en.wikipedia.org/wiki/Kitezh

Reply:
Ah OK. I guess it would be nice to see this files with a diff between vanilla and CK2+ changes.